Makefile.am

```diff
@@ -14,5 +14,6 @@ stubby.yml.windows: ${srcdir}/stubby.yml.example
 	awk "{sub(/$$/,\"\r\")}1" $< > $@
 
 install-data-hook: $(CONFIG_FILE)
-	test -f ${stubbyconfdir}/stubby.yml || \
-		${INSTALL_DATA} $(CONFIG_FILE) ${stubbyconfdir}/stubby.yml
+	$(INSTALL) -m 755 -d $(DESTDIR)$(stubbyconfdir)
```

Comment:
Not sure this line is necessary since we already have install the example file there normally.

Reply:
Excerpt from build log:
```
 /usr/bin/mkdir -p '/build/stubby/pkg/stubby/etc/stubby'
 /usr/bin/install -c -m 644 stubby.yml.example '/build/stubby/pkg/stubby/etc/stubby'
```

Reply:
(`DESTDIR=/build/stubby/pkg/stubby` in my case)